pypasta/MailCharacteristics.py

```diff
@@ -171,6 +171,18 @@ def _is_from_bot(self):
         if self.is_next and 'sfr@canb.auug.org.au' in email:
             return True
 
+        # Github Bot
+        if 'noreply@github.com' in email:
+            return True
+
+        # Buildroot's daily results bot
+        if '[autobuild.buildroot.net] Daily results' in subject:
```

Comment:
Please compare against subject.lower()

Reply:
this is not necessary since `subject = email_get_header_normalised(self.message, 'subject')` already lowers the subject.

Reply:
Oh, you're right - but then 'Daily results' will never match.